src/commands/cmd_stream.cc

```diff
@@ -358,6 +360,97 @@ class CommandXClaim : public Commander {
   }
 };
 
+class CommandAutoClaim : public Commander {
+ public:
+  Status Parse(const std::vector<std::string> &args) override {
+    CommandParser parser(args, 1);
+    key_name_ = GET_OR_RET(parser.TakeStr());
+    group_name_ = GET_OR_RET(parser.TakeStr());
+    consumer_name_ = GET_OR_RET(parser.TakeStr());
+    auto parse_int_status = parser.TakeInt<uint64_t>();
```

Comment:
I got your point, it's a valid reason. Having two variables is also a good decision. You can also try something like:
`if parse_status = ...; condition {`
I think C++17 offers this feature and there shouldn't be any shadowing.

Reply:
Hi @torwig,
Thanks for your suggestions.
I have updated the syntax with the `if-initialized` statement.

Best Regards,
Edward

Reply:
Hi @torwig ,

Thanks for your suggestions.
I have updated the syntax with the `if-initialized` statement.

Best Regards,
Edward